kale/sqs.py

```diff
@@ -41,12 +42,12 @@ def __init__(self, *args, **kwargs):
             if settings.MESSAGE_QUEUE_USE_PROXY:
                 # Used only in Dev environment which uses ElasticMQ instead of
                 # SQS. Hence it uses a different boto api to connect to a proxy
-                conn = boto.sqs.connection
-                self._connections[conn_str] = conn.SQSConnection(
-                    proxy_port=settings.MESSAGE_QUEUE_PROXY_PORT,
-                    proxy=settings.MESSAGE_QUEUE_PROXY_HOST, is_secure=False,
-                    aws_access_key_id=aws_access_key_id,
-                    aws_secret_access_key=aws_secret_access_key)
+                region = boto.sqs.regioninfo.RegionInfo(name='elasticmq', endpoint=settings.MESSAGE_QUEUE_PROXY_PORT)
```

Comment:
Name should not be elasticmq. Name it proxy maybe. There are many different possible proxies.

Reply:
Instead of MESSAGE_QUEUE_PROXY_PORT, this should be MESSAGE_QUEUE_PROXY_HOST

Reply:
Line too long, break it up after the (